backend/core/parser.py

```diff
@@ -0,0 +1,41 @@
+def parse_credit(text):
+    semester_name = {
+        'U000200001U000300001': '1',
+        'U000200001U000300002': 'S',
+        'U000200002U000300001': '2',
+        'U000200002U000300002': 'W',
+    }
+    lines = text.split('\n')[:-1]
+    columns = []
+    infos = []
+    for i, line in enumerate(lines):
+        line = line.split('\t')
+        if len(columns) != len(line):
+            columns = line
+            continue
+        dic = to_dict(columns, line)
+
+        # 26 : course taking info
+        # 13 : semester info
+        # 9 : total info
+        if len(dic) == 26:
```

Comment:
I think comparing `len(dic) == 26` is not good for code maintenance because it can be changed later.
I rather suggest condition like `if dic.haskey('ACQ_PNT')`.